example/50-seed.yaml

```diff
@@ -50,8 +50,8 @@ spec:
 # taints:
 # - key: seed.gardener.cloud/protected # only shoots in the `garden` namespace can use this seed
 # - key: <some-key>
-# DEPRECATED: The below keys are deprecated in favor of the `.spec.settings` section. Please don't use them anymore and
-# switch to the new seed settings.
+# DEPRECATED: The below keys are deprecated in favor of the `.spec.settings` section. They are bo longer respected and
```

Comment:
Imo we can remove the corresponding taints also from the example. The doc already mentions about the removed taints and points to the version in which they are removed.
`seed.gardener.cloud/protected` is still respected and still present, so we need to keep it.

Reply:
done